mopidy_pandora/library.py

```diff
@@ -161,11 +154,12 @@ def get_images(self, uris):
                     )
                 pass
             result[uri] = [models.Image(uri=u) for u in image_uris]
+        print(f"\n\n\nIMAGES: {result}\n\n\n")
```

Comment:
It might be better to use `logger`, with the appropriate level, here that this information is sent to the relevant log file.

Reply:
Ha, one of my temporary learning/debugging statements got left in, I think this should be removed completely